roles/openshift_openstack/templates/heat_stack.yaml.j2

```diff
@@ -226,6 +226,40 @@ resources:
 {% endif %}
 
 {% if not openshift_openstack_provider_network_name %}
+
+{% if not openshift_openstack_node_subnet_name %}
```

Comment:
why it is not enough with checking openshift_openstack_provider_network_name?

Reply:
got it! I also see that you limited the if openshift_openstack_provider_network_name block to just the net/subnet creation, which seems right. Thanks!